invesalius/data/viewer_volume.py

```diff
@@ -1294,22 +1297,22 @@ def CreateArrowActor(self, startPoint, endPoint):
         return actor_arrow
 
     def CenterOfMass(self):
-        barycenter = [0.0, 0.0, 0.0]
+
         proj = prj.Project()
         try:
             surface = proj.surface_dict[0].polydata
         except KeyError:
             print("There is not any surface created")
-            return barycenter
-        n = surface.GetNumberOfPoints()
-        for i in range(n):
-            point = surface.GetPoint(i)
-            barycenter[0] += point[0]
-            barycenter[1] += point[1]
-            barycenter[2] += point[2]
-        barycenter[0] /= n
-        barycenter[1] /= n
-        barycenter[2] /= n
+
```

Comment:
You must return the barycenter if except happen. Please add again the "barycenter = [0.0, 0.0, 0.0]" and "return barycenter"